src/components/popover-container/popover-container-content.component.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CloseIcon from './close-icon.style';
+import PopoverContainerContentStyle from './popover-container-content.style';
+import Events from '../../utils/helpers/events/events';
+import PopoverContainerHeaderStyle from './popover-container-header.style';
+import Title from './title.style';
+
+const PopoverContainerContent = ({
+  onClose, children, title, position
+}) => {
+  const handleCloseKeyDown = (ev) => {
+    if (Events.isEnterOrSpaceKey(ev)) {
+      ev.preventDefault();
+      onClose();
+    }
+
+    return null;
+  };
+
+  return (
+    <PopoverContainerContentStyle position={ position }>
+      <PopoverContainerHeaderStyle>
+        <Title>
+          {title}
+        </Title>
+        <CloseIcon
+          type='close'
+          tabIndex='0'
+          onClick={ onClose }
+          onKeyDown={ handleCloseKeyDown }
+        />
+      </PopoverContainerHeaderStyle>
+      {children}
+    </PopoverContainerContentStyle>
+  );
+};
+
+PopoverContainerContent.propTypes = {
+  onClose: PropTypes.func.isRequired,
```

Comment:
How does addon docs generate prop tables? Do we need to have comments for each prop here so they can be used in the table description field?